pages/PageDonation/[id]/[id2].vue

```diff
@@ -1,5 +1,8 @@
 <script setup lang="ts">
```

Comment:
give more descriptive names - we have two ids, but which id corresponds to what object?